version.json

```diff
@@ -1,4 +1,4 @@
 {
   "$schema": "https://raw.githubusercontent.com/dotnet/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
-  "version": "0.1"
+  "version": "0.2"
```

Comment:
Done to keep the minor version consistent with `Azure.Bicep.Types`, but this is not strictly necessary and can be backed out if desired.